### PR TITLE
Componentize explorer

### PR DESCRIPTION
**Pull request by jordanCain**

### CODE REVIEW

File: scripts/explorer-jes-configure.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# Copyright IBM Corporation 2019
+################################################################################
+
+################################################################################
+# Variables required on shell:
+# - STATIC_DEF_CONFIG_DIR
+# - ROOT_DIR
+# - NODE_HOME
+
+. $ROOT_DIR/scripts/utils/validateNode.sh
```
Comment (stevenhorsman) [inline scripts/explorer-jes-configure.sh]: This should be in component validate.sh, or the run-zowe.sh, not in component configure.sh

File: scripts/explorer-jes-start.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# - ROOT_DIR
+# - ZOWE_EXPLORER_HOST
+# - GATEWAY_PORT
+# - JES_EXPLORER_UI_PORT
+# - KEYSTORE_KEY
+# - KEYSTORE_CERTIFICATE
+# - KEYSTORE_PASSWORD
+
+# Find node binary
+. $ROOT_DIR/scripts/utils/validateNode.sh
```
Comment (stevenhorsman) [inline scripts/explorer-jes-start.sh]: This should be in component validate.sh, or the run-zowe.sh, not in component configure.sh

File: scripts/explorer-jes-configure.sh (hunk 1 above)
Comment (NakulManchanda) [reply scripts/explorer-jes-configure.sh]: creating explorer-jes-validate.sh

File: scripts/explorer-jes-start.sh (hunk 2 above)
Comment (NakulManchanda) [reply scripts/explorer-jes-start.sh]: creating explorer-jes-validate.sh, not checking here just assuming its there and building node bin
NODE_BIN=${NODE_HOME}/bin/node

File: scripts/explorer-jes-validate.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright IBM Corporation 2019
+################################################################################
+
+################################################################################
+# Variables required on shell:
+# - STATIC_DEF_CONFIG_DIR
+# - ROOT_DIR
+# - NODE_HOME
+
+. $ROOT_DIR/scripts/utils/validateNode.sh
```
Comment (stevenhorsman) [inline scripts/explorer-jes-validate.sh]: Can you rename this to validate-node please to match the rest of the script naming convention (sorry if I confused by only fixing some of the others recently)

Comment (NakulManchanda) [reply scripts/explorer-jes-validate.sh]: renamed

**Pull request merged**